raiden/network/proxies/token_network.py

```diff
@@ -223,27 +273,44 @@ def detail_participant(
             'nonce': data[4],
         }
 
-    def detail_channel(self, participant1: typing.Address, participant2: typing.Address) -> Dict:
-        """ Returns a dictionary with the channel specific information. """
+    def detail_channel(
+            self,
+            participant1: typing.Address,
+            participant2: typing.Address,
+            channel_identifier: typing.Optional[typing.ChannelID] = None,
```

Comment:
When setting the default to `None` you don't need to write `Optional[XXX]`.

Reply:
I know I know, but I find it a lot more readable when the code is explicit

Reply:
Fine for me!